.rubocop.yml

```diff
@@ -17,18 +17,19 @@ Lint/ImplicitStringConcatenation:
     - 'lib/generators/sufia/**/*'
 
 Metrics/LineLength:
-  Max: 400
+  Enabled: false
 
 Metrics/AbcSize:
-  Max: 30
-  Exclude:
-    - 'app/controllers/concerns/sufia/admin/depositor_stats.rb'
-    - 'app/controllers/concerns/sufia/my_controller_behavior.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
+  Enabled: false
```

Comment:
I haven't ever run into this complexity metric (or at least, I've never tripped over it). Is it giving a lot of false positives or what?